TestingDangerApp/ViewController.swift

```diff
@@ -1,3 +1,11 @@
+//
+//  AppDelegate.swift
+//  TestingDangerApp
+//
+//  Created by PicPay Eduardo on 7/23/19.
+//  Copyright © 2019 PicPay Eduardo. All rights reserved.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Edu15/TestingDangerApp/blob/24dcd26e92c19401966aad13f7c897ac4d2b5a2b/TestingDangerApp/ViewController.swift#L6"></span>Header comments should be consistent with project patterns.<br />
<code>file_header</code> <code>ViewController.swift:6</code></td>
    </tr>
  </tbody>
</table>